### PR TITLE
Hd/update wizard to handle quality asset

### PR DESCRIPTION
**Pull request by RSlysz**
### Purpose of this PR
Update of Wizard to better handle assets in Quality Settings
Fixed issue due to missing resource folder.
Fixed issue where DefaultVolumeProfile was checked against currentAsset instead of defaultAsset
Added checks on Global Settings
Remove the useless Populate/Reset (not really usefull since we don't have the custom scenes and duplicate with fixes under)
Updated documentation and added missing DXR update on documentation.
Added localization on each fields.
RenderPipelineWizard1

---
### Testing status
Most of the tests have been done in prior 10.5 and 2021.1 branch for this update.

---
### Comments to reviewers
Separate PR will also target:
- 10.5: https://github.com/Unity-Technologies/Graphics/pull/3932 
- 2021.1/staging: https://github.com/Unity-Technologies/Graphics/pull/3974

@JordanL8 be cautious, most of the change in the doc is the same than other 2 PR but there is an additional check on **Assigned - HDRP Settings**

@jenniferd-unity please thorougly check at the change I had to do in the ched/fix methods regarding global settings and GraphicSettings to see if i had not do some strange stuff.

### CODE REVIEW

**Review by remi-chapelain — APPROVED** [review]
Did a quick smoke test (Fixing wizard on HDRP, HDRP + DXR), changed quality settings before and after fixing to make sure it retrieve the proper asset. ✔️ 

The rest has already been checked on [this backport PR ](https://github.com/Unity-Technologies/Graphics/pull/3932)

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md [hunk 1]
```diff
@@ -37,31 +27,45 @@ There are three tabs that you can use to set up your HDRP Project for different
 * [HDRP + VR](#VRTab): Use this tab to set up your HDRP Project and enable support for virtual reality.
 * [HDRP + DXR](#DXRTab): Use this tab to set up your HDRP Project and enable support for ray tracing.
 
+Also every configuration is separated in two scopes:
+
+- **Global:** settings in Unity Editor or in the [HDRP Asset](HDRP-Asset.md) set in Graphics Settings.
```
Comment (jenniferd-unity) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md:32]: Could be changed to say
`settings in Unity Editor, [HDRP Global Settings](Default-Settings-Window.md), or Graphics Settings'  [HDRP Asset](HDRP-Asset.md)`

**Review by jenniferd-unity — APPROVED** [review]
Small suggestion for the doc

**Review by sebastienlagarde — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md [hunk 2]
```diff
@@ -37,31 +27,45 @@ There are three tabs that you can use to set up your HDRP Project for different
...
-| **- Diffusion Profile**          | Checks to make sure that your HDRP Asset references a [**Diffusion Profile**](Diffusion-Profile.md) Asset.<br />Press the **Fix** button to reload the runtime resources for the HDRP Asset. |
-| **Default Volume Profile** | Checks to make sure you have assigned a **Default Volume Profile Asset** in **Edit** > **Project Settings** > **HDRP Default Settings** .<br />Press the **Fix** button to open a pop-up that allows you to either assign a Profile or create and assign a new one. |
+| **Assigned - Graphics** | Checks to make sure you have assigned an [HDRP Asset](HDRP-Asset.md) to the **Graphics Settings** field (menu: **Edit** > **Project Settings** > **Graphics**).<br />Press the **Fix** button to open a pop-up that allows you to either assign an HDRP Asset or create and assign a new one. |
+| **Assigned - HDRP Settings** | Checks to make sure you have assigned an **HDRenderPipelineGlobalSettings** to the **HDRP Settings** field (menu: **Edit** > **Project Settings** > **HDRP Graphics**).<br />Press the **Fix** button will try to find first one available. If no one is available, it will create one in the **Default Resources Folder**. |
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md:50]: Recommended change:

| **Assigned - Graphics** | Checks to make sure you have assigned an **HDRenderPipelineGlobalSettings** asset to the **HDRP Settings** field (menu: **Edit** > **Project Settings** > **HDRP Graphics**).<br />Press the **Fix** button to find and assign an available **HDRenderPipelineGlobalSettings** asset. If there isn't one available, Unity creates an **HDRenderPipelineGlobalSettings** in the **Default Resources Folder**.

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md [hunk 3]
```diff
@@ -37,31 +27,45 @@ There are three tabs that you can use to set up your HDRP Project for different
...
+| **SRP Batcher**          | Checks to make sure that SRP Batcher is enabled.<br />Press the **Fix** button to enable it in the used HDRP Asset. |
 
 <a name="VRTab"></a>
 
 ### HDRP + VR
 
-This tab provides all of the configuration options from the [HDRP tab](#HDRPTab) as well as extra configuration options to help you set your HDRP Project up to support virtual reality. If you can not find an option in this section of the documentation, check the [HDRP tab](#HDRPTab) above. This is only supported on Windows OS.
+This tab provides all of the configuration options from the [HDRP tab](#HDRPTab) as well as extra configuration options to help you set your HDRP Project up to support virtual reality. If you can not find an option in this section of the documentation, check the [HDRP tab](#HDRPTab) above. This is only supported on Windows OS. The extra configuration are all in the **Global** scope.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md:68]: Recommended change: 

You can adjust the extra configuration options in the  **Global** scope.

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Made some suggestions, please check them for technical accuracy.

**Review by Vic-Cooper — APPROVED** [review]
Thanks for making the changes! No further suggestions :)

**Pull request merged**